include/wx/qt/private/winevent.h

```diff
@@ -327,6 +330,126 @@ class wxQtEventSignalHandler : public Widget, public wxQtSignalHandler< Handler
     virtual bool winEvent ( MSG * message, long * result ) { }
     virtual bool x11Event ( XEvent * event ) { } */
 
+    virtual bool event(QEvent *event)
+    {
+        if (event->type() == QEvent::Gesture){
```

Comment:
It would be great if you could please use wx code style and put braces on their own lines, i.e.
```suggestion
        if (event->type() == QEvent::Gesture)
        {
```
(here and below).

Reply:
Agreed. Will fix.